source/solvers/heat_transfer_assemblers.cc

```diff
@@ -465,6 +465,9 @@ HeatTransferAssemblerRobinBC<dim>::assemble_rhs(
             this->boundary_conditions_ht.Tinf[i_bc]->value(Point<dim>());
           const double emissivity =
             this->boundary_conditions_ht.emissivity[i_bc]->value(Point<dim>());
+          const double heat_flux_bc =
+            this->boundary_conditions_ht.heat_flux_bc[i_bc]->value(
+              Point<dim>());
```

Comment:
```suggestion

```
The heat flux BC is seperate from convection-diffusion, so you should remove it from there.